packages/aws-cdk-lib/package.json

```
@@ -61,12 +59,11 @@
         "maven": {
           "groupId": "software.amazon.awscdk",
           "artifactId": "lib",
```

```suggestion
          "artifactId": "lib"
```

Think this is what's breaking the build.

---

Gah!!